tools/cldr-apps/WebContent/js/CldrSurveyVettingTable.js

```diff
@@ -325,19 +325,52 @@ const cldrSurveyTable = (function() {
 	 * AND by insertFixInfo in review.js (Dashboard "Fix")!
 	 */
 	function updateRow(tr, theRow) {
+		if (!tr || !theRow) {
+			return;
+		}
+		if (tr.checksum !== undefined) {
```

Comment:
minor.

Move const newChecksum = cldrChecksum(JSON.stringify(theRow)); above the undefined loop, call it "rowChecksum" (and change references below).

Reply:
Good idea. I'll make another PR.